hummingbot/connector/connector/uniswap_v3/uniswap_v3_connector.py

```diff
@@ -55,11 +58,28 @@ def __init__(self,
     def name(self) -> str:
         return "uniswap_v3"
 
-    def get_order_price_quantum(self, trading_pair: str, price: Decimal) -> Decimal:
-        return Decimal("1e-6")
-
-    def get_order_size_quantum(self, trading_pair: str, order_size: Decimal) -> Decimal:
-        return Decimal("1e-6")
+    async def initiate_pool(self) -> str:
+        """
+        Initiate connector and start caching paths for trading_pairs
+        """
+        while True:
+            try:
+                # self.logger().info(f"Initializing Uniswap connector and paths for {self._trading_pairs} pairs.")
+                resp = await self._api_request("get", "eth/uniswap/v3/start",
+                                               {"pairs": json.dumps(self._trading_pairs)})
+                status = bool(str(resp["success"]))
+                if status:
```

Comment:
If the status is false, this loop can keep on sending API requests indefinitely, is this intended? Shouldn't an exception be raised instead?

Reply:
Requests to this endpoint don't trigger a direct request to the the network. The gateway uses this like a keep-alive just like the way keep-alive works in websocket.